documents/customer.md

```diff
@@ -155,6 +155,249 @@ Customer customer = client.Customer.Fetch("cust_M462yViJrhNrQc");
+customerRequest.Add("beneficiary_mobile", "8762489310");
+customerRequest.Add("beneficiary_city", "Bangalore");
+customerRequest.Add("beneficiary_state", "KA");
+customerRequest.Add("beneficiary_country", "IN");
+
+BankAccount refund = client.Customer.Fetch(customerId).DeleteBankAccount(customerRequest);
+```
+
+**Parameters:**
+
+| Name                 | Type     | Description                                                   |
+|----------------------|----------|---------------------------------------------------------------|
+| customerId*          | string   | Unique identifier of the customer.                            |
+| account_number       | string   | Customer's bank account number.                               |
+| beneficiary_name     | string   | The name of the beneficiary associated with the bank account. |
+| beneficiary_address1 | string   | The virtual payment address.                                  |
+| beneficiary_email    | string   | Email address of the beneficiary.                             |
+| beneficiary_mobile   | integer  | Mobile number of the beneficiary.                             |
+| beneficiary_city     | string   | The name of the city of the beneficiary.                      |
+| beneficiary_state    | string   | The state of the beneficiary.                                 |
+| beneficiary_pin      | interger | The pin code of the beneficiary's address.                    |
+| ifsc_code            | string   | The IFSC code of the bank branch associated with the account. |
+
+**Response:**
+```json
+{
+  "id" : "cust_1Aa00000000001",
+  "entity": "customer",
+  "name" : "Saurav Kumar",
+  "email" : "Saurav.kumar@example.com",
+  "contact" : "+919000000000",
+  "gstin":"29XAbbA4369J1PA",
+  "notes" : [],
+  "created_at ": 1234567890
+}
+```
+
+-------------------------------------------------------------------------------------------------------
+
+### Delete Bank Account of Customer
+
+```C#
+string customerId = "cust_N5mywh91sXB69O"
+
+String bankAccountId = "ba_N6aM8uo64IzxHu"
+
+Customer refund = client.Customer.Fetch(customerId).DeleteBankAccount(bankAccountId);
+```
+
+**Parameters:**
+
+| Name          | Type        | Description                                 |
+|---------------|-------------|---------------------------------------------|
+| customerId*   | string      | Unique identifier of the customer.          |
+| bankAccountId | string      | The bank_id that needs to be deleted.       |
+
+**Response:**
+```json
+{
+    "id": "ba_Evg09Ll05SIPSD",
+    "ifsc": "ICIC0001207",
+    "bank_name": "ICICI Bank",
+    "name": "Test R4zorpay",
+    "account_number": "XXXXXXXXXXXXXXX0434",
+    "status": "deleted"
+}
+```
+
+-------------------------------------------------------------------------------------------------------
+
+### Eligibility Check API
+
+```C#
+Dictionary<string, object> customerRequest = new Dictionary<string, object>();
+customerRequest.Add("inquiry","affordability");
```

Comment:
I am not checking the schema of all these APIs since I don't have context on these. Please make sure these are correct.

Reply:
I had tested this Dictionary while testing api